### ISSUE TITLE
Test on different datasets

### ISSUE THREAD

**Issue by melhashash** [prs-eth/ImpliCity#1]
Hello,

Thanks for open-sourcing the project. I am wondering if you can provide a simple example/steps on how to test on different datasets, since there are some parameters to define?

**Comment by markkua** [comment]
Hello, thanks for your question.

We've provided a demo with pretrained model weights and demo data (one 64m x 64m patch). To run the demo, please refer to [this instruction](https://github.com/prs-eth/ImpliCity#demo). 

To apply ImpliCity to other datasets, you first need to format the dataset as our [demo data](https://github.com/prs-eth/ImpliCity#download-demo). All the data needs to be cropped to the same spatial extent. The input point cloud and query points should be saved as `.npz` files and the images (ortho-photos, masks) as `.tif` rasters. I strongly suggest loading our demo data and checking their format and content. Regarding the sampling and occupancy determination of query points, you can refer to [the script](https://github.com/prs-eth/ImpliCity/blob/main/scripts/dataset_ImpliCity/build_dataset.py) I used to generate our training data. 

Regarding training settings (and parameters), you may want to start with our default settings specified in these [configuration files](https://github.com/prs-eth/ImpliCity/tree/main/config/train_test).

I hope you find this answer helpful. Please do not hesitate to contact us again in case of follow-up questions.
